### PR TITLE
Feature api post cart

### PR DESCRIPTION
**Pull request by codenuk**
add page /admin/product

redirect page /admin/login to /admin

connect api page /myorder, /admin/myorder, /member

redirect page /checkout to /customize

and some css page

### CODE REVIEW

**Review by cl21484952 — DISMISSED** [review]

File: src/components/admin-login/index.js [hunk 1]
```diff
@@ -17,15 +17,14 @@ const AdminLoginComponent = () => {
...
-                </h3>
+                <h3>Stickerwish Admin Login</h3>
                 <label>อีเมล<ErrorMessage name="email" render={msg => <span className="error">{msg}</span>} /></label>
                 <Field name="email" type="email" className={styles.inputText} placeholder="" />
 
                 <label>รหัสผ่าน<ErrorMessage name="password" render={msg => <span className="error">{msg}</span>} /></label>
                 <Field name="password" type="password" className={styles.inputText} placeholder="" />
-
+                
+                {values.checkLogin && <p className="error" style={{ marginBottom: "10px"}}>username หรือ password ไม่ถูกต้อง</p>}
```
Comment (SorawitSiri) [inline src/components/admin-login/index.js]: Should not use Inner-line

File: src/components/admin-order/index.js [hunk 2]
```diff
@@ -11,10 +12,27 @@ import GroupDeliveryPayment from "../group-delivery-payment";
...
-    const { values } = useFormikContext();
-    // const [dropDawn, setDropDawn] = useState(0);
+    const { values, setFieldValue } = useFormikContext();
     const [selectStep] = useState(3);
-    const [expandCard, setExpandCard] = useState(0);
+
+    useEffect(() => {
+        axios.get(`https://asia-east2-digitalwish-sticker.cloudfunctions.net/orders`)
+          .then(res => {
+            console.log("res.data[0]", res.data[0])
```
Comment (SorawitSiri) [inline src/components/admin-order/index.js]: You should comment this line.

File: src/components/approve-layout/index.js [hunk 3]
```diff
@@ -10,53 +11,70 @@ import GroupDeliveryPayment from "../group-delivery-payment";
...
-            setFieldValue("itemsList", orderNumber.itemsList);
-        }
-    };
-
+    const [guestMode] = useState(false);
+
+    useEffect(() => {
+        axios.get(`https://asia-east2-digitalwish-sticker.cloudfunctions.net/orders`)
+          .then(res => {
+            console.log("res.data[0]", res.data[0])
```
Comment (SorawitSiri) [inline src/components/approve-layout/index.js]: THIS HERE!!

File: src/components/group-delivery-payment/index.js [hunk 4]
```diff
@@ -1,84 +1,97 @@
...
+                                <td className={styles.textRight}>{listCard.units}</td>
+                                <td className={styles.textRight}>{listCard.price}฿</td>
+                            </tr>
+                        )})
+                    }
+                        </tbody>
+                        <tbody>
+                            <tr>
+                                <td colspan="3" style={{ borderTop: "1px solid black" }}>ค่าสินค้ารวม</td>
+                                <td className={styles.textRight} style={{ borderTop: "1px solid black" }}>{values.itemsCost}฿</td>
```
Comment (SorawitSiri) [inline src/components/group-delivery-payment/index.js]: YOU should not use style inner-line

File: src/components/admin-order/index.js (hunk 2 above)
Comment (codenuk) [reply src/components/admin-order/index.js]: commented

File: src/components/approve-layout/index.js (hunk 3 above)
Comment (codenuk) [reply src/components/approve-layout/index.js]: commented

File: src/components/home-member/index.js [hunk 5]
```diff
@@ -9,25 +10,34 @@ import { withFormik } from 'formik';
 import { useFormikContext } from 'formik';
 
 const LabelSatus = ({ status }) => {
```
Comment (SorawitSiri) [inline src/components/home-member/index.js:13]: You should use
```
STATUS_TYPE = {
    DOING: "กำลังดำเนินการ",
    xxxx: "กำลังผลิตสินค้า",
    xxx: "อยู่ระหว่างจัดส่ง",
    xxx: "ขอคืนเงิน",
    xxx: "คืนเงินสำเร็จ",
    xx: "รายการสำเร็จ"
}
```

**Review by SorawitSiri — APPROVED** [review]

**Pull request merged**